src/easyscience/Fitting/minimizers/engine_lmfit.py

```diff
@@ -2,36 +2,34 @@
 #  SPDX-License-Identifier: BSD-3-Clause
 #  © 2021-2023 Contributors to the EasyScience project <https://github.com/easyScience/EasyScience
 
-__author__ = "github.com/wardsimon"
-__version__ = "0.1.0"
+__author__ = 'github.com/wardsimon'
+__version__ = '0.1.0'
 
 import inspect
+from typing import Callable
 from typing import List
 from typing import Optional
 
+import numpy as np
 from lmfit import Model as lmModel
-
-# Import lmfit specific objects
 from lmfit import Parameter as lmParameter
 from lmfit import Parameters as lmParameters
 from lmfit.model import ModelResult
 
-from easyscience.Fitting.fitting_template import Callable
-from easyscience.Fitting.fitting_template import FitError
-from easyscience.Fitting.fitting_template import FitResults
-from easyscience.Fitting.fitting_template import FittingTemplate
-from easyscience.Fitting.fitting_template import NameConverter
-from easyscience.Fitting.fitting_template import np
+from .minimizer_base import MinimizerBase
+from .utils import FitError
+from .utils import FitResults
+from .utils import NameConverter
 
 
-class lmfit(FittingTemplate):  # noqa: S101
+class LmFit(MinimizerBase):  # noqa: S101
     """
     This is a wrapper to lmfit: https://lmfit.github.io/
```

Comment:
The correct link is https://lmfit.github.io/lmfit-py/
The current link gives a 404